sw/device/silicon_creator/lib/manifest.h

```diff
@@ -22,9 +22,10 @@ extern "C" {
  *
  * OpenTitan secure boot, at a minimum, consists of four boot stages: ROM,
```

Comment:
We can relax the requirement as some silicon owners may opt for not using `BL0`.

Reply:
Done.